template.rb

```diff
@@ -197,6 +199,11 @@ def get_gh_file_content(filename)
 
   # copy database.yml sample
   FileUtils.cp "config/database.yml.sample", "config/database.yml"
+
+  # copy .env.sample if the .env is missing
+  unless File.exist?(".env")
+    FileUtils.cp ".env.sample", ".env"
```

Comment:
I think we are crrently not creating the `.env.sample` file (would be empty) and we are not adding the `.env` file to the `.gitignore` file.

I don't know if that should be fixed in this PR though, maybe it's something for another story? what do you think

Reply:
Okay adding it to another story in JIRA.

Reply:
https://ombulabs.atlassian.net/browse/DT-101 & https://ombulabs.atlassian.net/browse/DT-102